packages/suite/src/views/wallet/coinmarket/buy/components/BuyForm/Inputs/index.tsx

```diff
@@ -214,24 +217,56 @@ const Inputs = () => {
 
                                 if (amountLimits) {
                                     const amount = Number(value);
-                                    if (amountLimits.minCrypto && amount < amountLimits.minCrypto) {
+
```

Comment:
Could you take the whole `register` callback out from the render and into the component body? Would be much easier to read.

Reply:
Done.

Reply:
perfect, thanks!